be/src/olap/rowset/segment_reader.h

```diff
@@ -308,7 +308,7 @@ class SegmentReader {
 
     // If this field is false, client must to call seek_to_block before
     // calling get_block.
-    bool _at_block_start = false;
+    bool _seek_block = true;
```

Comment:
Field don't match comment. And I think _at_block_start name is easier to understand.